src/encoded/static/components/cart/cart.js

```diff
@@ -1606,17 +1621,21 @@ export const processPseudoDefaultFiles = (datasets = []) => {
                     // from all visualizable files.
                     const outputTypes = visualizableFiles.reduce((accOutputTypes, file) => accOutputTypes.add(file.output_type), new Set());
                     if (outputTypes.size > 1) {
-                        // Multiple output types among the files. Group first by replicate and then
-                        // by output_type.
-                        const replicateFiles = _(visualizableFiles).groupBy((file) => file.biological_replicates[0]);
-                        Object.keys(replicateFiles).forEach((replicate) => {
-                            const outputTypeFiles = _(replicateFiles[replicate]).groupBy((file) => file.output_type);
-                            Object.keys(outputTypeFiles).forEach((outputType) => {
-                                // Select the first bigWig and first bigBed within each
-                                // output_type within each replicate.
-                                const firstBigWig = outputTypeFiles[outputType].find((file) => file.file_format === 'bigWig');
-                                const firstBigBed = outputTypeFiles[outputType].find((file) => file.file_format === 'bigBed');
-                                pseudoDefaultFiles.push(..._.compact([firstBigWig, firstBigBed]));
+                        // Multiple output types among the files. Group first by sorted assembly,
```

Comment:
This looks fine. Philosophically, since you have one part feeding another part, you can use underscore.js chain. But it may be overkill to some.